code/datums/commodity.dm

```diff
@@ -1024,6 +1024,15 @@
 	upperfluc = 500
 	lowerfluc = -500
 
+/datum/commodity/contraband/syndicate_headset
+	comname = "Illegal Headset"
+	comtype = /obj/item/device/radio/headset/syndicate
+	desc = "This headset allows you to speak over a highly illegal Syndicate frequency."
+	price = 500
```

Comment:
500 is not 750 - intentional?

Reply:
I don't recall which way it was intended to be so i'm sticking with 500